docker/compose/data-restore/docker-compose.yaml

```diff
@@ -3,36 +3,36 @@
 version: "3.8"
 services:
   restore:
-    image: aptoslabs/tools:devnet
+    image: aptoslabs/tools:nightly
     volumes:
       - type: volume
         source: db
         target: /opt/aptos/data
       - type: volume
         source: tmp
         target: /tmp
+      # Depends on which cloud backup data you use, replace this with either:
+      # `s3.toml` (AWS S3)
+      # `gcs.toml` (GCP GCS)
       - type: bind
-        source: ./s3-public.yaml
-        target: /opt/aptos/etc/s3-public.yaml
```

Comment:
is `s3.yaml` the same as `s3-public.yaml` now? unathenticated public access bucket?

Reply:
Probably leave a small note saying that this will restore data from Aptos Labs public backup